tools/generate_test_data.Rmd

```diff
@@ -21,7 +21,9 @@ knitr::opts_chunk$set(
 )
 if (!require(NRTAPmodel)) {
   pak::pak("Spatiotemporal-Exposures-and-Toxicology/NRTAPmodel@isong-calc-covars")
+  pak::pak("Spatiotemporal-Exposures-and-Toxicology/amadeus")
```

Comment:
Old username GitHub paths?

Reply:
Fixed the path and cleaned redundant lines.